.github/workflows/main.yml

```diff
@@ -3,63 +3,105 @@ name: Python package
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
+    - name: Install miniconda
+      uses: goanpeca/setup-miniconda@v1
+      with:
+        auto-update-conda: true
+        activate-environment: sunode-dev
+        python-version: ${{ matrix.python-version }}
+        channels: conda-forge,defaults
+        channel-priority: strict
     - name: Install Dependences
+      shell: bash -l {0}
       run: |
-        export PATH="/usr/share/miniconda/bin:$PATH"
-        conda create -n sunode-env -q --yes -c conda-forge python=${PYVER} conda-build conda-verify coverage pytest hypothesis statsmodels
-      env:
-        PYVER: ${{ matrix.python-version }}
+        conda info
+        conda install --yes conda-build conda-verify coverage pytest hypothesis statsmodels
+    - name: Log conda and Python versions
+      shell: bash -l {0}
+      run: |
+        conda --version
+        conda info
+        which python
+        python --version
+        conda config --get
+    - name: Build package
+      shell: bash -l {0}
+      run: |
+        conda build --variants "{python: [${{ matrix.python-version }}]}" ./conda
+    - name: Log conda and Python versions again
+      shell: bash -l {0}
+      run: |
+        conda --version
+        which python
+        python --version
+        conda config --get
+        ls /usr/share/miniconda/envs
+        ls /usr/share/miniconda/envs/sunode-dev
+        ls /usr/share/miniconda/envs/sunode-dev/conda-bld
+        ls /usr/share/miniconda/envs/sunode-dev/conda-bld/noarch/
     - name: Install package 
+      shell: bash -l {0}
       run: |
-        export PATH="/usr/share/miniconda/bin:$PATH"
-        source activate sunode-env
-        conda-build -c conda-forge ./conda
-        conda install --yes -c file://${CONDA_PREFIX}/conda-bld/ -c conda-forge sunode
+        conda install --yes -c file:///usr/share/miniconda/envs/sunode-dev/conda-bld sunode
+    - name: debug
+      shell: bash -l {0}
+      run: |
+        conda info
     - name: Test with coverage
+      shell: bash -l {0}
       run: |
-        export PATH="/usr/share/miniconda/bin:$PATH"
-        source activate sunode-env
-        python -c "import sunode"
         coverage run -m pytest --pyargs sunode
-  - name: Windows Build
+  test-on-windows:
     runs-on: ${{ matrix.operating-system }}
     strategy:
       max-parallel: 4
       matrix:
         operating-system: [windows-latest]
-        python-version: [3.8]
+        python-version: [3.7]
     steps:
-    - uses: actions/checkout@v1
-    - name: Install conda build dependences
-      run: |
-        conda install -c conda-forge conda-build conda-verify
-    - name: Build sundials locally
-      run: |
-        git clone https://github.com/conda-forge/sundials-feedstock
-        conda build -c conda-forge ./sundials-feedstock/recipe
-    - name: Build sunode
+    - uses: actions/checkout@v2
+    - name: Debug dir
+      shell: cmd /C call {0}
```

Comment:
@aseyboldt Where does `{0}` come from? Or is this some kind of shell-switching?

Reply:
That makes sure that we really get a properly activated conda env. See https://github.com/marketplace/actions/setup-miniconda#IMPORTANT
Took me ages to find that...